.gitignore

```diff
@@ -6,3 +6,11 @@ node_modules/
 rvm-installer
 rvm-installer.asc
 *.log
+
+.DS_Store
+**/.DS_Store
```

Comment:
Won't the previous line be enough?

Reply:
no actually in a PR i saw someone any how created the file in includes/ so thats why.

Reply:
I add `.DS_Store` to master's `.gitignore` and then created a `.DS_Store` it's ignored. So I don't think you need the `**/.DS_Store` variant.

Reply:
Thank you for investigation. Removed it :)